libs/ledger-live-common/src/families/evm/buildOptimisticOperation.ts

```diff
@@ -0,0 +1,38 @@
+import eip55 from "eip55";
+import { Account, Operation } from "@ledgerhq/types-live";
+import { encodeOperationId } from "../../operation";
+import { Transaction as EvmTransaction } from "./types";
+import { getEstimatedFees } from "./logic";
+
+/**
+ * Create a temporary operation to use until it's confirmed by the blockchain
+ */
+export const buildOptimisticOperation = (
+  account: Account,
+  transaction: EvmTransaction
+): Operation => {
+  const type = "OUT";
+  const estimatedFees = getEstimatedFees(transaction);
+  const value = transaction.amount.plus(estimatedFees);
+
+  // keys marked with a <-- will be updated by the broadcast method
+  const operation: Operation = {
```

Comment:
I guess using a` Partial<Operation>` wouldn't work ?

Reply:
You're right that it would make more sense, I just kept using our old way of doing this. Let me check if that would work typing wise cause I'm not sure yet if the optimistic operation isn't stored in a `Operation[]` somewhere before reaching the broadcast 👀

Reply:
Yeah that's what I thought, unfortunetaly changing it to a `Partial<Operation>` would not work with the return type of `signOperation`: `Observable<SignOperationEvent>`. I could still change it, but that would mean decoupling this integration from the others making it harder to debug and overall complexify our code base. 

We would definitely keep it in mind though and we should discuss making this change for all integrations if possible, as a general improvement 👍